jujugui/static/gui/src/app/components/deployment/add-credentials/add-credentials.js

```diff
@@ -32,14 +34,30 @@ YUI.add('deployment-add-credentials', function() {
       @method _handleCloudClick
     */
     _handleAddCredentials: function(id) {
-      this.props.changeState({
-        sectionC: {
-          component: 'deploy',
-          metadata: {
-            activeComponent: 'summary'
+      // Add template
+      var user = this.props.users.jem.user,
```

Comment:
Not that there will be different templates for the three public clouds. Please add a XXX TODO here for the future, so that this will be AWS specific, but we'll need to handle GCE & Azure as well.